docusaurus/src/theme/EditThisPage/index.tsx

```diff
@@ -0,0 +1,24 @@
+// Swizzle action eject is unsafe to perform on EditThisPage.
+// It is more likely to be affected by breaking changes in the future
+// https://docusaurus.io/docs/swizzling/
+//
+// We added nofollow to the link for SEO purposes.
```

Comment:
suggestion

It's prolly just me but I almost missed this line because my mind thought it was an import:

```suggestion
// We added nofollow to the link for SEO purposes.

```